packages/store/src/index.js

```diff
@@ -1,8 +1,11 @@
-export { makeStore } from './store.js';
-export { makeWeakStore } from './weak-store.js';
+export { makeScalarMap, makeScalarMap as makeStore } from './scalarMap.js';
+export {
+  makeScalarWeakMap,
+  makeScalarWeakMap as makeWeakStore,
+} from './scalarWeakMap.js';
+export { makeLegacyMap } from './legacyMap.js';
+export { makeLegacyWeakMap } from './legacyWeakMap.js';
+
 export { makeExternalStore } from './external/default.js';
 export { makeMemoryExternalStore } from './external/memory.js';
 export { makeHydrateExternalStoreMaker } from './external/hydrate.js';
-
-// Backward compatibility.
-export { makeStore as default } from './store.js';
```

Comment:
Will this break any dapps? Do we need to announce this as a breaking change?

Reply:
Good point. I now export `makeLegacyMap as default`, with an explanatory comment. Should help avoid breakage. Resolving.